src/store/genomicSuggesterTypes.ts

```diff
@@ -0,0 +1,57 @@
+import { ThunkDispatch } from 'redux-thunk';
+import { RootState } from './rootState';
+import { SearchText, SelectedSuggestion, Suggestion } from './graphql/variants/models';
+
+export const TOGGLE_LOADING = 'TOGGLE_LOADING_GENOMIC_SUGGESTER';
+export const FAILURE = 'FAILURE_GENOMIC_SUGGESTER';
+export const RE_INITIALIZE_STATE = 'RE_INITIALIZE_STATE_GENOMIC_SUGGESTER';
+export const CLEAR_SUGGESTIONS = 'CLEAR_MESSAGE_GENOMIC_SUGGESTER';
+export const ADD_SUGGESTIONS = 'ADD_SUGGESTIONS_GENOMIC_SUGGESTER';
+export const SELECT_SUGGESTION = 'SELECT_SUGGESTION_GENOMIC_SUGGESTER';
+
+export type SelectedSuggestionAction = {
+  type: typeof SELECT_SUGGESTION;
+  selectedSuggestion: SelectedSuggestion;
+};
+
+interface AddSuggestions {
+  type: typeof ADD_SUGGESTIONS;
+  suggestions: Suggestion[];
+  searchText: SearchText;
+}
+
+interface Failure {
+  type: typeof FAILURE;
+  error: Error | null;
+}
+
+interface ReInitializeState {
+  type: typeof RE_INITIALIZE_STATE;
+}
+
+interface ClearSuggestions {
+  type: typeof CLEAR_SUGGESTIONS;
+}
+
+interface ToggleLoading {
+  type: typeof TOGGLE_LOADING;
```

Comment:
aren't all your type 'string' if you use `typeof` ? isn't a enum will be clearer in that case? Why is it a interface and not a type? I can't see any object implanting the interface.